cluster/charts/crossplane/README.md

```diff
@@ -78,7 +78,7 @@ and their default values.
 | `extraVolumesCrossplane` | Add custom `volumes` to the Crossplane pod. | `{}` |
 | `hostNetwork` | Enable `hostNetwork` for the Crossplane deployment. Caution: enabling `hostNetwork`` grants the Crossplane Pod access to the host network namespace. | `false` |
 | `image.pullPolicy` | The image pull policy used for Crossplane and RBAC Manager pods. | `"IfNotPresent"` |
-| `image.repository` | Repository for the Crossplane pod image. | `"crossplane/crossplane"` |
+| `image.repository` | Repository for the Crossplane pod image. | `"xpkg.upbound.io/crossplane/crossplane"` |
```

Comment:
@plumbis I think I'll need to open a docs PR to update https://github.com/crossplane/docs/blob/master/content/master/software/install.md - there's no automation keeping that file in sync with this README.md, is there?

Reply:
@plumbis confirmed that a separate manual PR is needed 🤓